### ISSUE TITLE
AI auto complete in message editing

### ISSUE THREAD

**Issue by micromaomao** [micromaomao/chat.maowtm.org#6]

**Comment by 837951602** [comment]
![PZ{T55E}EQE@ X@FN$KU}%Y](https://github.com/micromaomao/chat.maowtm.org/assets/24836316/7a46a85a-2581-4824-a602-e2d65fde9a66)
Is this auto-complete leak?

**Comment by micromaomao** [comment]
yeah, thanks for reporting... Although I don't know a better way of dealing with it other than detecting these cases and handling them specially.
